pymchelper/run.py

```diff
@@ -120,7 +127,8 @@ def main(args=sys.argv[1:]):
         if parsed_args.output is not None:
             output_dir = os.path.dirname(parsed_args.output)
             if output_dir and not os.path.exists(output_dir):
-                raise IOError("Directory {}/ does not exist.".format(output_dir))
+                os.makedirs(output_dir)
+                logger.warning("Directory {:s} does not exist, creating.".format(output_dir))
```

Comment:
I think logger should go first in case `os.makedirs()` fails and errors, but I don't remember if it will end program execution or not.

Reply:
I agree, I'll fix it.